catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/endpoints.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<!--
+=============================================================
+Endpoints.xml
+Services (HTTP/WS) that catalog-ui-search provides for the UI
+=============================================================
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:jaxrs="http://cxf.apache.org/blueprint/jaxrs">
+
+    <!--
+    =============================================================
+    SIMPLE APPS (AUTH/BUILD/CATALOG/PLATFORM/FEEDBACK/GEOCODER/LOGOUT/SESSION/USER)
+    DESCRIPTORS APPLICATION
+    LIST APPLICATION
+    METACARD APPLICATION
+    QUERY APPLICATION
+    SEARCH FORMS APPLICATION
+    =============================================================
+    -->
+
+    <service ref="sparkServlet" interface="javax.servlet.Servlet">
+        <service-properties>
+            <entry key="urlPatterns" value="/search/catalog/internal/*"/>
+            <entry key="async-supported" value="true"/>
+        </service-properties>
+    </service>
+
+    <service ref="socketServlet" interface="javax.servlet.Servlet">
+        <service-properties>
+            <entry key="urlPatterns" value="/search/catalog/ws"/>
+            <entry key="async-supported" value="true"/>
+        </service-properties>
+    </service>
+
+    <!--
+    =============================================================
+    PROXYING
+    =============================================================
+    -->
+    <!-- TODO Proxy stuff that might be related to the proxy data in the config app, but not sure -->
```

Comment:
❓ Could use some background on the proxy servlet. I had trouble tracing the proxy data in the configuration app back to this servlet. Are they related or do we have multiple user stories regarding proxying expressed here?

Reply:
There is only one user story here. It relates to proxying for map layers and by leveraging Camel, the proxy can remain restrictive based upon configuration and not act as a generic, wide-open proxy. Leveraging existing Camel components also saves us from implementing our own, configurable proxy to accomplish the same goal.